front/app/containers/MainHeader/DesktopNavbar/AdminPublicationsNavbarItem.tsx

```diff
@@ -0,0 +1,241 @@
+import React, { useState, useEffect, MouseEvent, FormEvent } from 'react';
```

Comment:
The Projects item in the desktop navbar, just pulled it out of the existing navbar and refactored a bit.